inc/servervars.php

```diff
@@ -32,9 +32,10 @@
     require_once __DIR__ . '/../cdn/deploy/cdn.php';
   }
 
-  $site_suffix = KeymanHosts::Instance()->Site_Suffix();
+  $site_suffix = KeymanHosts::Instance()->Tier() == KeymanHosts::TIER_DEVELOPMENT ?
+    ".localhost" : "";
```

Comment:
@Meng-Heng - I think this bit is why our local Docker setup wasn't working.

`KeymanHosts::Instance()->Site_Suffix()` was refactored out in the newer shared-sites version

Reply:
Thank you! I pulled the latest changes from s.keyman.com and deleted `KeymanHosts::Instance()->Site_Suffix()`, and now it works almost perfectly. 
![image](https://github.com/user-attachments/assets/017768e3-47d1-466f-b6f2-4860d4038b6e)